packages/core/src/Provider/TxSubmitProvider/types.ts

```diff
@@ -6,7 +6,7 @@ type SerializedTransaction = Cardano.util.HexBlob;
 
 export interface SubmitTxArgs {
   signedTransaction: SerializedTransaction;
-  context?: { handles: HandleResolution[] };
+  context?: { handleResolutions: HandleResolution[] };
```

Comment:
@rhyslbw this refactor makes sense but we need to be careful with deployments: since the name changed, old backend will not validate when you pass it as `handles:`. Also I noticed we don't have `context` specified in `openApi.json` - should we add it in this PR?

Reply:
Good points.

1. The breaking change will need to be reverted (deferred)
2. Yes, adding the missing OpenAPI spec content is a good idea here

Reply:
Ok, refactoring will be moved to separate PR

Reply:
Captured openApi.json task in LW-7535